packages/ui-informed/src/form/Form.tsx

```diff
@@ -0,0 +1,83 @@
+import { cn } from '@utima/ui';
+import { Form as InformedForm } from 'informed';
+import { useMemo, type ComponentProps } from 'react';
+import { useTranslation } from 'react-i18next';
+import type { ZodObject, ZodRawShape } from 'zod';
+
+import { FormContext, type FormContextType } from '../hooks/useFormContext';
+import {
+  useFormSubmitActions,
+  type TypedFormState,
+} from '../hooks/useFormSubmitActions';
+
+export type FormProps<T> = Omit<
+  ComponentProps<typeof InformedForm>,
+  'initialValues' | 'onSubmit'
+> & {
+  initialValues: T;
+  disabled?: boolean;
+  readOnly?: boolean;
+  loading?: boolean;
+  columns?: boolean;
+  onSubmit?: (formState: TypedFormState<T>) => Promise<unknown>;
+  zodSchema?: ZodObject<ZodRawShape>;
+};
+
+/**
+ * Basic form component used to wrap form inputs. It provides action bar,
+ * column layout, and wraps all content in Informed Form component,
+ * which handles form state and validation.
+ */
+export function Form<T>({
+  onSubmit,
+  children,
+  initialValues,
+  readOnly = false,
+  disabled = false,
+  loading = false,
+  columns = true,
```

Comment:
I would omit columns entirely, this is app specific, so can be done easily by wrapping this component in the application (or rendering columns in different components)